datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validators/validator.py

```diff
@@ -251,9 +251,50 @@ def check_sysobjectids_are_duplicated(self, sysobjectids, profile):
                 self.duplicated[sysobjectid] = self.used_sysobjid[sysobjectid]
 
 
+class TableColumnHasTagValidator(ProfileValidator):
+    """
+    Validator responsible to check if all metrics related to a column in a profile
+    have tags to identify which row/entity they belong to
+    """
+
+    def validate(self, profile, path):
+        file_contents = find_profile_in_path(profile, path)
+        metrics = {}
+        if file_contents.get('metrics'):
+            metrics = file_contents.get('metrics')
```

Comment:
If that's a dictionary, this is shorter:
```suggestion
        metrics = file_contents.get('metrics', {})
```